test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -285,6 +322,20 @@ public void wipeCluster() throws Exception {
         adminExecutionContext.callApi("snapshot.delete_repository", deleteSnapshotsArgs, Collections.emptyList(), Collections.emptyMap());
     }
 
+    /**
+     * @return An exclude list of indices that will not be removed in between tests.
+     */
+    protected List<String> excludeIndices() {
```

Comment:
naming nit: maybe call this persistentIndices ? something that tells that these indices will survive the before/after test. exclude doesn't say what they are excluded from.